VEDA_OS01/tests/utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Utils for tests.
+"""
+from VEDA_OS01.models import TranscriptCredentials
```

Comment:
I think `utils` should avoid to import models. Instead you can pass model to decorator something like 
```python
def invalidate_fernet_cached_properties(model):
   ...
   field = model._meta.get_field(field_name)
```

and then, you can call the util method

`invalidate_fernet_cached_properties(TranscriptCredentials)`

Reply:
This looks good, I will change this